### PR TITLE
registration get cached based on variantId and channel

### PR DESCRIPTION
**Pull request by edewit**
fixing [AGWIN-1](https://issues.jboss.org/browse/AGWIN-1)

### CODE REVIEW

File: aerogear-windows-push/IChannelStore.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using System;
```
Comment (matzew) [inline aerogear-windows-push/IChannelStore.cs]: license headers?

File: aerogear-windows-push/IChannelStore.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AeroGear.Push
+{
+    public interface IChannelStore
+    {
```
Comment (matzew) [inline aerogear-windows-push/IChannelStore.cs]: API docs?

**Comment by matzew** [comment]
@sebastienblanc can you review ?

**Comment by sebastienblanc** [comment]
Tested and working.
Once @matzew  comments are processed I'm :+1:

**Pull request merged**